Add WebDriver conformance section (#25930)
mdn/content: files/en-us/mozilla/firefox/releases/112/index.md

#### WebDriver BiDi

- Implemented `browsingContext.print` as PDF command. See [Firefox bug 1806810](https://bugzil.la/1806810) for more details
- Implemented `script.addPreloadScript` and `script.removePreloadScript` commands. See [Firefox bug 1806420](https://bugzil.la/1806420) and [Firefox bug 1806465](https://bugzil.la/1806465) for more details

> - Element and ShadowRoot references (NodeCache) now have the same `sharedId` in both Marionette and WebDriver BiDi, making it possible to use them in both protocol ([Firefox bug 1770733](https://bugzil.la/1770733)).

Reviewer: ```suggestion
- `Element` and `ShadowRoot` references as stored in the node cache can now be used n both Marionette and WebDriver BiDi by their exact same unique reference. See [Firefox bug 1770733](https://bugzil.la/1770733) for more details.
```

> - Removed `isRedirect` from the network events base parameters ([Firefox bug 1819875](https://bugzil.la/1819875)).

Author: I wanted to be a bit lighter for the simpler changes

> - Removed `isRedirect` from the network events base parameters ([Firefox bug 1819875](https://bugzil.la/1819875)).

Reviewer: If that's fine for the MDN content reviewers then it's fine for me as well.

#### WebDriver BiDi

- Implemented `browsingContext.print` as PDF command. See [Firefox bug 1806810](https://bugzil.la/1806810) for more details
- Implemented `script.addPreloadScript` and `script.removePreloadScript` commands. See [Firefox bug 1806420](https://bugzil.la/1806420) and [Firefox bug 1806465](https://bugzil.la/1806465) for more details
- Element and ShadowRoot references (NodeCache) now have the same `sharedId` in both Marionette and WebDriver BiDi, making it possible to use them in both protocol ([Firefox bug 1770733](https://bugzil.la/1770733)).
- Removed `isRedirect` from the network events base parameters ([Firefox bug 1819875](https://bugzil.la/1819875)).

#### Marionette

- Fixed an issue happening on response wrapped in`value` field ([Firefox bug 1819029](https://bugzil.la/1819029)).

> -  Fixed an issue where `WebDriver:ElementClear` was emitting an extra `change` event for content editable elements ([Firefox bug 1744925](https://bugzil.la/1744925)).

Reviewer 2: ```suggestion
- Fixed an issue where `WebDriver:ElementClear` was emitting an extra `change` event for content editable elements ([Firefox bug 1744925](https://bugzil.la/1744925)).
```

> 

Reviewer 2: ```suggestion
```
